test/odh/notebook.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package odh
```

Comment:
Would that make sense to have that file merged into the `support.go` one?

Reply:
well...
First thing is that the name itself, `support.go`, is not much descriptive (against clean code rules). I kept it there as the same name is used everywhere else.
Also `support.go` is dedicated for providing resource content. Having it also Notebook code seems to like too much extending the scope of that file.

Reply:
Make sense.